devel/cquery/Portfile

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       waf 1.0 xcodeversion 1.0
+
+name            cquery
+
+categories      devel
+license         MIT
+maintainers     {wyuenho @wyuenho} openmaintainer
+description     C/C++/Objective C Language Server Protocol support
+long_description \
+cquery is a highly-scalable, low-latency language server for \
+C/C++/Objective-C. It is tested and designed for large code bases like \
+Chromium. cquery provides accurate and fast semantic analysis without \
+interrupting workflow. \
+\
+cquery implements almost the entire language server protocol and provides some \
+extra features.
+
+platforms       darwin freebsd
+homepage        https://github.com/cquery-project/cquery
+
+version         v20180302
+revision        0
+
+fetch.type      git
+git.url         https://github.com/cquery-project/cquery
+git.branch      ${version}
```

Comment:
Should we be using the `github` portgroup for this?

Reply:
There are no buildable tarballs on the their github releases, nor any usable binaries. This port requires pulling submodules, a quick look around the github portgroup file doesn't seem to give me any help in that regard, am I right?

Reply:
That may in fact be true, though it's unfortunate if it is. For now lets merge this, and why don't you ask about the submodule issue on macports-dev?

Reply:
Did you try using the GitHub Portgroup? I see no reason why it wouldn't work. The project has releases. We also don't want the "v" at the beginning of the version number. The GitHub portgroup correctly handles that.

Reply:
For subports, you can call the GitHub.setup separately for each port.